docs/source/development/traffic_router/traffic_router_api.rst

```diff
@@ -41,32 +41,32 @@ Request Structure
 
 	GET /crs/stats HTTP/1.1
 	Host: trafficrouter.infra.ciab.test
-	User-Agent: curl/7.47.0
+	User-Agent: curl/7.52.1
 	Accept: */*
 
 Response Structure
 ------------------
 .. code-block:: http
-	:caption: Response Example
+	:caption: Response Example (JSON expanded)
 
-	HTTP/1.1 200 OK
+	HTTP/1.1 200
```

Comment:
Though the HTTP spec doesn't actually care whether or not the reason-phrase exists, our lexer does. They're not accepting pull requests, trust me I've tried.
So basically, this needs to be `HTTP/1.1 200 OK` not just `HTTP/1.1 200`.

Reply:
The reason-phrase was missing because I was sending HTTP requests and, for the API, [sendReasonPhrase is only set for HTTPS](https://github.com/apache/trafficcontrol/commit/9838c0e0d6#diff-383ce4b8e55d3e7e25a66bbe5b5f317eR40-R45). Is this PR a good place to add it for HTTP, or does that warrant its own PR?

Reason-phrase re-added in 23d18b71.